src/bindgen/mod.rs

```diff
@@ -67,3 +67,4 @@ pub use self::builder::Builder;
 pub use self::config::Profile; // disambiguate with cargo::Profile
 pub use self::config::*;
 pub use self::error::Error;
+pub use self::library::Library;
```

Comment:
I'm confused, why does this help? Don't you need `unstable_ir` as well to make this useful? Should this be part of the `unstable_ir` feature?